browser/src/DownloadsPage/ResearchDownloads.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+
+import { Badge, ExternalLink, Link as StyledLink, List, ListItem } from '@gnomad/ui'
+
+import Link from '../Link'
+
+import {
+  FileList,
+  GenericDownloadLinks,
+  GetUrlButtons,
+  ShowURLButton,
+  IndexedFileDownloadLinks,
+  SectionTitle,
+} from './downloadsPageStyles'
+
+const ResearchDownloads = () => {
+  return (
+    <>
+      <p>
+        The research data set contains data from various research projects that are relevant to
+        gnomAD, and the datasets are not curated by the gnomAD Production Team. Each dataset has an
+        accompanying README file explaining what it contains.
+      </p>
+      <section>
+        <SectionTitle id="v3-genomic-constraint">Non-coding and Genomic constraint</SectionTitle>
+        <p>For more information about these files, see the README included in the download.</p>
+        {/* TODO:(rgrant) link to blog post when this is released too */}
+        {/* <p>
+          For more information about these files, see our blog post on{' '}
+          <ExternalLink href="/">the addition of short tandem repeat calls to gnomAD</ExternalLink>.
+        </p> */}
+        <FileList>
+          {/* @ts-expect-error TS(2745) FIXME: This JSX tag's 'children' prop expects type 'never... Remove this comment to see the full error message */}
+          <ListItem>
+            <GetUrlButtons
+              label="All files"
+              gcsBucket="gnomad-nc-constraint-v31-paper"
+              path="/download_files/"
+              includeAWS={false}
+              includeAzure={false}
+            />
+          </ListItem>
+          {/* @ts-expect-error TS(2745) FIXME: This JSX tag's 'children' prop expects type 'never... Remove this comment to see the full error message */}
+          <ListItem>
+            <GetUrlButtons
+              label="README"
+              gcsBucket="gnomad-nc-constraint-v31-paper"
+              path="/download_files/nc_constraint_gnomad_v31_README.docx"
+              includeAWS={false}
+              includeAzure={false}
+            />
+          </ListItem>
+          {/* @ts-expect-error TS(2745) FIXME: This JSX tag's 'children' prop expects type 'never... Remove this comment to see the full error message */}
+          <ListItem>
+            <GetUrlButtons
```

Comment:
Currently when I try to download the files here i get a bucket storage path. This is probably not ideal for most people... they likely prefer to download them directly from the browser.

![image](https://user-images.githubusercontent.com/8867295/197021174-f270d4a1-d9b7-475a-932a-49a7bfc5a6ac.png)

If you use a URL like `https://storage.googleapis.com/gnomad-nc-constraint-v31-paper/download_files/nc_constraint_gnomad_v31_README.docx`, people can just download without using `gsutil`. Not sure if this would work for the "All files" option though.

Reply:
Aha, very good to know. Thanks!